WeakAuras/Modernize.lua

```diff
@@ -2074,6 +2074,39 @@ function Private.Modernize(data, oldSnapshot)
     end
   end
 
+  if data.internalVersion < 76 then
+    local function removeHoles(t)
+      local keys = {}
+      for key in pairs(t) do
+        table.insert(keys, key)
+      end
+      table.sort(keys)
+      if #keys ~= #t then
+        local newTable = {}
+        for i, key in ipairs(keys) do
+          newTable[i] = t[key]
+        end
+        return newTable
+      else
+        return t
+      end
+    end
+
+    for _, triggerData in ipairs(data.triggers) do
+      local trigger = triggerData.trigger
+      if trigger.type and trigger.event and Private.category_event_prototype[trigger.type] then
```

Comment:
We can't use category_event_prototype/event_prototype in Modernize, because they will change as triggers evolve, but the Modernize should not change.

Reply:
fixed both comments